app/src/main/java/org/mozilla/fenix/settings/CustomizationFragment.kt

```diff
@@ -135,17 +133,6 @@ class CustomizationFragment : PreferenceFragmentCompat() {
         addToRadioGroup(topPreference, bottomPreference)
     }
 
-    private fun setupHomeCategory() {
-        requirePreference<PreferenceCategory>(R.string.pref_home_category).apply {
-            isVisible = FeatureFlags.topFrecentSite
-        }
-        requirePreference<SwitchPreference>(R.string.pref_key_enable_top_frecent_sites).apply {
-            isVisible = FeatureFlags.topFrecentSite
-            isChecked = context.settings().showTopFrecentSites
-            onPreferenceChangeListener = SharedPreferenceUpdater()
```

Comment:
I think we still need to keep this SharedPreferenceUpdater() bit to make sure it's updating the value in our `Settings`

Reply:
Omg thank you